tsconfig.json

```diff
@@ -20,6 +20,7 @@
     "experimentalDecorators": true,
     "emitDecoratorMetadata": true,
     "esModuleInterop": true,
+    "typeRoots": ["./@types", "node_modules/@types"]
```

Comment:
Isn't the default behavior for typeRoots already included all @types folder including under node_module? Is it just to be more explicit?

Reply:
Yes, in this case it might be overkill. Do you do want me to remove this explicit configuration?

Reply:
I don't think so. Explicit is always better.